tapeout/src/main/scala/transforms/macros/Utils.scala

```diff
@@ -0,0 +1,154 @@
+// See LICENSE for license details.
+
+package barstools.tapeout.transforms.macros
+
+import firrtl._
+import firrtl.ir._
+import firrtl.PrimOps
+import firrtl.Utils.{ceilLog2, BoolType}
+import scala.util.parsing.json.JSON // Todo: this will be gone
+import java.io.File
+import scala.language.implicitConversions
+
+trait PortPolarity
+case object ActiveLow extends PortPolarity
+case object ActiveHigh extends PortPolarity
+case object NegativeEdge extends PortPolarity
+case object PositiveEdge extends PortPolarity
+object PortPolarity {
+  implicit def toPortPolarity(s: Any): PortPolarity =
```

Comment:
This has to be Any for the implicit to work? Do you ever call it with something other than a String?